### DISCUSSION TITLE
filtering criteria and multiplets

### DISCUSSION THREAD

**Discussion by tanasa** [stuart-lab/signac#1118] [General]
Dear authors, when we apply the filtering criteria in Seurat, 

which criteria can we  add/ modify in such a way that we can make sure that we exclude the multiplets ?

```
pbmc <- (
  x = pbmc,
  subset = peak_region_fragments > 3000 &
    peak_region_fragments < 20000 &
    pct_reads_in_peaks > 15 &
    blacklist_ratio < 0.05 &
    nucleosome_signal < 4 &
    TSS.enrichment > 2
)
```

In ArchR, calling the doublets/multiplets is based on synthetic doublets (https://www.archrproject.com/bookdown/how-does-doublet-identification-work-in-archr.html).

How shall we do it in Signac ? Thank you.

**Comment by timoast** [comment]
You can try doublet finding methods designed for scRNA-seq such as scrublet, DoubletFinder, etc.

Alternatively, you can try AMULET which was designed for scATAC-seq data and uses the number of sites with >2 reads to identify doublets.

**Comment by BartBryant** [comment]
@timoast , have you used AMULET, DoubletFinder, or scrublet? As AMULET is made for scATACseq datasets, it is 'better'? 
if one does use DoubletFinder, do we first make the Gene Activity Matrix first using GeneActivity(  )? Add the gene activity to the Seurat Object with 'RNA' modality and use this object as input into DoubletFinder?

**Reply by timoast** [reply]
I have not used AMULET but it makes sense if it may perform better for scATAC-seq. In the AMULET paper they have some comparison with ArchR which uses a more conventional synthetic doublet approach

**Comment by AC-H** [comment]
Does anyone have a pipeline to apply both libraries (Signac and ArchR)?
Thanks in advance.
Kind regards,
Ana.
